app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -33,7 +40,9 @@ class StateFragmentPresenter @Inject constructor(
 
   private var showCellularDataDialog = true
   private var useCellularData = false
-
+  private var llRoot: LinearLayout? = null
+  private var dummyEditTextButton: Button? = null
+  private var dummyFetchDataTV: TextView? = null
```

Comment:
You can use `lateinit` with these three views to avoid `null` value.

Example: `private lateinit var dummyFetchDataTV: TextView`

Reply:
resolved